### DISCUSSION TITLE
Home assistant in docker e command_line

### DISCUSSION THREAD

**Discussion by Gerrino65** [indomus/forum#1180] [HUB: Home Assistant]
Ciao. Dopo una migrazione da HA Core a docker con qualche problema risolto grazie ai consigli trovati qui, mi ritrovo a fare qualche finizione. Magari un'occasione per imparare qualcosa. Vengo al problemino attuale.
Tempo fa ho installato un package che utilizza ripetutamente l'integrazione command-line. In un caso nei log registra un errore. si tratta di "command: "cat /home/homeassistant/.homeassistant/.HA_VERSION"".
Ho fatto diverse prove aggiungendo sudo e cambianto prorietario del file, senza successo. Penso (sbagliato?) dipenda dal fatto che il file .HA_VERSION si trova in quella directory. E' un problema di poco conto ma mi piacerebbe risolvere. Chi mi aiuta?

**Comment by snakuzzo** [comment]
Quello che hai scritto non funzionerà mai.
Probabilmente non ti è chiaro come funziona Docker.
Quei comandi verranno eseguiti all'interno del container...seguendo la struttura directory all'interno del container.
La directory di configurazione di HA nel container è `/config`.
Quindi il comando sarà...

```
cat /config/.HA_VERSION
```

P.S. `sudo` non è che si usa a caso

**Reply by Gerrino65** [reply]
Infatti ho accennato al dubbio che il problema fosse legato alla collocazione del file .HA_VERSION. Quello che mi confondeva è che da shell "cat /home/homeassistant/.homeassistant/.HA_VERSION" come utente pi dava il risultato aspettato e nessun errore. Grazie

**Reply by snakuzzo** [reply]
Infatti quel comando lo dai da `pi` sull'host
Invece HA lo lancia all'interno del container...quindi devi rispettare quella struttura di directory
